libraries/pbrlib/genglsl/mx_generalized_schlick_bsdf.glsl

```diff
@@ -20,7 +20,7 @@ void mx_generalized_schlick_bsdf_reflection(vec3 L, vec3 V, vec3 P, float occlus
     float avgAlpha = mx_average_alpha(safeAlpha);
     vec3 Ht = vec3(dot(H, X), dot(H, Y), dot(H, N));
 
-    FresnelData fd = mx_init_fresnel_schlick(color0, color90, exponent);
+    FresnelData fd = mx_init_fresnel_schlick_airy(color0, color90, exponent, bsdf.thickness, bsdf.ior);
```

Comment:
Just a nit-picky comment on code structure: In the dielectric case we check for the presence of thin-film here, and call the right fresnel init function accordingly. In the generalized schlick case now the airy fresnel version is always called and the check for thin-film is done inside the init instead. I don't have a strong opinion on which approach is better, but for consistency and readability we could use the same approach in all BSDF's.

Very minor, and we can fix this in another change list.